pubsub/cloud-client/publisher_test.py

```diff
@@ -43,6 +45,19 @@ def topic(client):
     yield topic_path
 
 
+def _make_sleep_patch():
```

Comment:
You should use the `monkeypatch` fixture built in to pytest. The reason is that that fixture will automatically clean up any patched functions.

Reply:
Then again, I see you are using this in a context manager, which has the same effect. Nevermind.

Reply:
Ohh that's good to know!